packages/fxa-payments-server/Gruntfile.js

```diff
@@ -0,0 +1,48 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+'use strict';
+
+module.exports = function (grunt) {
+  grunt.initConfig({
+    pkg: grunt.file.readJSON('package.json'),
+    concat: {
+      ftl: {
+        src: [
+          'src/lib/l10n/branding.ftl',
+          'src/*.ftl',
+          'src/lib/*.ftl',
+          'src/routes/**/*.ftl',
+          'src/components/**/*.ftl',
+        ],
+        dest: 'public/locales/en/payments.ftl',
+      },
+
+      // We need this for tests because we pull the latest from `fxa-content-server-l10n`
+      // and place those in our `public` directory at `postinstall` time, and sometimes we have
+      // FTL updates on our side that haven't landed yet on the l10n side. We want to test
+      // against _our_ latest, and not necessarily the l10n repo's latest.
+      'ftl-test': {
+        src: ['src/**/*.ftl'],
+        dest: 'test/payments.ftl',
+      },
+    },
+    watch: {
+      ftl: {
+        files: ['src/**/*.ftl'],
+        tasks: ['merge-ftl'],
+        options: {
+          interrupt: true,
+        },
+      },
+    },
+  });
+
+  grunt.loadNpmTasks('grunt-contrib-watch');
+  grunt.loadNpmTasks('grunt-contrib-concat');
+
+  grunt.registerTask('merge-ftl', ['concat:ftl']);
+  grunt.registerTask('merge-ftl:test', ['concat:ftl-test']);
+  grunt.registerTask('watch-ftl', ['watch:ftl']);
```

Comment:
For this to watch properly, you have to update the `pm2` config file, which contains these kind of watch processes.

Reply:
Thanks for pointing that out, I'll include that in the updated commit.